### PR TITLE
A (very) quick and dirty OSX compatibility

### PR DESCRIPTION
**Pull request by yonicozac**
Hi, 

Doing plugins like you mentioned would be better, but this is better than nothing :)

Tested on Lion only.

### CODE REVIEW

File: myip/myip_cmd.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 # Consts
 IP_BIN = "/sbin/ip"
 BLACKLIST_IFACES = "lo"
-INTERFACE_NAME_PREFIX_BY_PRIORITY = ["eth", "wlan"]
+INTERFACE_NAME_PREFIX_BY_PRIORITY = ["eth", "wlan", "en0", "en1"]
```
Comment (yonicozac) [inline myip/myip_cmd.py:16]: en0 - OSX ethernet
en1 - OSX wifi

File: myip/myip_cmd.py [hunk 2]
```diff
@@ -88,20 +115,32 @@ def parse_args(args):
...
+    else:
+        logging.error("Platform %s is not supported", sys.platform)
+        return []
+
     if config.interface: # interface param provided
         return [get_ip_of_specific_interface(config.interface)]
 
     ifaces_hash = parse_ip_addr_cmd()
-    filtered_ifaces = [k for k in ifaces_hash.keys() if k not in BLACKLIST_IFACES]
+    filtered_ifaces = [k for k in ifaces_hash.keys() if not k.startswith(BLACKLIST_IFACES)]
```
Comment (yonicozac) [inline myip/myip_cmd.py:143]: So support lo0 etc etc

**Comment by orenhe** [comment]
Sweet!!
I will get to merge it soon and pluginize, so it'll auto choose the proper plugin.

**Comment by orenhe** [comment]
Hi, I've made some refactoring to make it more modular. I also added unittests for Darwin output, but I don't have a real MacOS setup :)

Can you pull latest and check it out?

10x

**Comment by yonicozac** [comment]
Ok, seems to work fine.
thanks!.

**Pull request merged**